cmd/staticProxy.go

```diff
@@ -92,15 +90,19 @@ func runProxyBlocking() {
 		addHeadersToContext = append(addHeadersToContext, []byte(value))
 	}
 
-	prox := http.NewFastStaticProxy(http.FastStaticProxyConfig{
+	backendURL, err := url.Parse(staticProxyBackendURL)
+	if err != nil {
+		panic(err)
+	}
+
+	prox := fastproxy.NewFastStaticProxy(fastproxy.FastStaticProxyConfig{
 		MiddleWares: []middleware.GraphqlMiddleware{
 			&middleware.ValidationMiddleware{},
 			&middleware.ContextMiddleware{},
 		},
 		RequestConfigProvider: proxy.NewStaticSchemaProvider(proxy.RequestConfig{
 			Schema:              &schema,
-			BackendAddr:         []byte(staticProxyBackendURL),
-			BackendHost:         staticProxyBackendAddr,
+			BackendURL:          *backendURL,
```

Comment:
Curious as to your opinion on this, but I couldn't find a reason to keep the addr and url separate. Using the stdlib url.URL type, and using the access methods on it as needed.